site/docs/v1/tech/identity-providers/index.adoc

```diff
@@ -76,4 +95,143 @@ An email address or domain may be provided in the `login_hint` request parameter
 
 You can read more about the `login_hint` and `idp_hint` parameters in the link:/docs/v1/tech/oauth/endpoints/[OAuth Endpoints documentation].
 
-include::docs/v1/tech/identity-providers/_linking-strategies.adoc[]
+== Linking Strategies
+
+image::identity-providers/linking-strategy.png[Linking Strategies, role=top-cropped bottom-cropped, width=1200]
+
+The linking strategy to use when creating the link between the Identity Provider and the user.
+
+The possible values are:
+
+* `Create a Pending Link` - Do not automatically link, instead return a pending link identifier that can be used to link to an existing user.
+* `Anonymously Link` - Always create a link based upon the unique Id returned by the identity provider. A username or email is not required and will not be used to link the user. A reconcile lambda will not be used in this configuration.
+* `Link On Email` - Link to an existing user based upon email. A user will be created with the email returned by the identity provider if one does not already exist.
+* `Link On Email. Do not create the user if they do not exist` - Only link to an existing user based upon email. A user will not be created if one does not already exist with email returned by the identity provider.
+* `Link On Username` - Link to an existing user based upon username. A user will be created with the username returned by the identity provider if one does not already exist.
+* `Link On Username. Do not create the user if they do not exist` - Only link to an existing user based upon username. A user will not be created if one does not already exist with the username returned by the identity provider.
+
+Here's a table illustrating the alternatives:
+
+[cols="1,1,1,3"]
+|===
+| Strategy | User must exist in FusionAuth | User linked on | Use when the identity provider
+
+| Create a Pending Link | Yes | Yes, user must login to existing account | Has a different email or username than an existing FusionAuth identity and users know enough to link them.
+| Anonymously Link | No | IdP id | Exposes neither username nor email.
+| Link On Email | No | Email address | Has the user's email and users that do not exist in the identity provider can have access.
+| Link On Email. Do not create the user if they do not exist | Yes | Email address | Has the user's email and you don't want users that do not exist in the identity provider to have access.
+| Link On Username | No | Username | Has the user's username and users that do not exist in the identity provider can have access.
+| Link On Username. Do not create the user if they do not exist | Yes | Username | Has the user's username and you don't want users that do not exist in the identity provider to have access.
```

Comment:
See above for thoughts

Reply:
> you don't want users that do not exist in the identity provider to have access.

in other words, I don't think that this statement is accurate.  Or I am really confused (either way 🙂 )